lib/ansible/module_utils/avi.py

```diff
@@ -0,0 +1,36 @@
+#
+# Created on December 12, 2016
+# @author: Gaurav Rastogi (grastogi@avinetworks.com)
+# Avi Version: 16.3.4
+#
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
```

Comment:
Just noticed the license here.  Generally, module_utils files are BSD licensed so that people writing custom modules can feel that they are okay to license their custom modules as they choose.